Libraries/ActionSheetIOS/ActionSheetIOS.js

```diff
@@ -25,6 +36,13 @@ var ActionSheetIOS = {
       typeof callback === 'function',
       'Must provide a valid callback'
     );
+
+    var processedTintColor = this.processTintColor(options.tintColor);
+
+    if (processedTintColor) {
+      options.tintColor = processedTintColor;
+    }
```

Comment:
I think this can just be collapsed to 

    options.tintColor = processColor(options.tintColor);